### ISSUE TITLE
How to exclude file types?

### ISSUE THREAD

**Issue by gaul** [musikid/pjdfstest#80]
For example, I don't care to test Block, Char, and Socket.

**Comment by asomers** [comment]
You can't.  There's no way to do that currently.  But I think there will be in the GSoC rewrite.

**Comment by musikid** [comment]
Yes, as @asomers said, this is currently not possible. However, tests which operates on a file type have the type appended to its name. We could add an `--invert-match` argument to then dismiss tests which end with `::block` or `::char` for example. Or maybe a dedicated argument like `--file-types` and then do the filtering transparently. What do you think?
